study-builder/studies/testboston/emails/kit_arrived.en.html

```diff
@@ -1,14 +1,16 @@
 #set( $text = {
-  "subject": "Your test kit was delivered!",
+  "subject": "Your kit will be delivered very soon",
   "salutation": "Dear {{participant_firstName}} {{participant_lastName}},",
-  "delivered": "Our records show your kit was just delivered. It’s time to fill in your monthly survey and take your COVID-19 test.",
-  "same_time": "It’s important to do the survey and test at about the same time.",
+  "p1": "Your COVID-19 test will be delivered very soon.",
+  "p2": "When you receive it, it is important to <span style=\"font-weight:bold\">collect your sample, complete your monthly survey, and return your test only Monday through Thursday</span> (tests returned over the weekend will be discarded).",
```

Comment:
I tried running this locally and it appears that the templating engine (Apache Velocity) doesn't like escaped double-quotes... Switching to single-quotes will work.